sys/random/prng_tinymt32.c

```diff
@@ -35,6 +36,21 @@ uint32_t random_uint32(void)
     return tinymt32_generate_uint32(&_random);
 }
 
+size_t random_bytes(uint8_t *buf, size_t size)
+{
+  size_t count = 0;
+  uint32_t last;
+
+  while(size > sizeof(uint32_t)) {
+    *((uint32_t*)(buf+count)) = tinymt32_generate_uint32(&_random);
```

Comment:
This code may (/will) cause alignment errors on ARM systems, as `alignof(uint8_t) <= alignof(uint32_t)`.

Reply:
I believe this should be fixed in the newest commit